entrypoint/restartservice/restartservice_test.go

```diff
@@ -0,0 +1,75 @@
+package restartservice
+
+import (
+	"fmt"
+
+	"github.com/rancher/distros-test-framework/pkg/assert"
+	"github.com/rancher/distros-test-framework/pkg/testcase"
+
+	. "github.com/onsi/ginkgo/v2"
+)
+
+var _ = Describe("Test:", func() {
+
+	It("Start Up with no issues", func() {
+		testcase.TestBuildCluster(GinkgoT())
+	})
+
+	It("Validate Nodes", func() {
+		testcase.TestNodeStatus(
+			assert.NodeAssertReadyStatus(),
+			nil,
+		)
+	})
+
+	It("Validate Pods", func() {
+		testcase.TestPodStatus(
+			assert.PodAssertRestart(),
+			assert.PodAssertReady(),
+			assert.PodAssertStatus(),
+		)
+	})
+
+	It("Restart server and agent service", func() {
+		testcase.TestRestartService()
+	})
+
```

Comment:
Re verifying the node and pod status here -after the restart, should be enough to ensure the cluster is actually up - before testing the cluster ip and other stuff -> this could be added here instead of adding a simple delay at the end of the restart test - without doing any verification.

Reply:
idk if i totally got your point, but if i do , just adding the tests after fails, because need some sec to cluster come back.

Reply:
after restart step, add these 2 tests again:
```
It("Validate Nodes", func() {
		testcase.TestNodeStatus(
			assert.NodeAssertReadyStatus(),
			nil,
		)
	})

	It("Validate Pods", func() {
		testcase.TestPodStatus(
			assert.PodAssertRestart(),
			assert.PodAssertReady(),
			assert.PodAssertStatus(),
		)
	})
```

the validation test for this, is in a loop, waiting for the node and pod status to come up.

Reply:
got youu!!!   tk you